posts/2022/esds-fall-event.md

```diff
@@ -0,0 +1,35 @@
+---
+author: Julia Kent
+date: 2022-10-10
+tags: tutorial hackathon events
+---
+
+# ESDS Fall Event
```

Comment:
We can use the title offered by Katie at the last ESDS committee meeting: 

A (Re)Introduction to Earth System Data Science (ESDS) Across NCAR/UCAR/UCP

unless there is an alternative from the folks?